coding-concepts.js

```diff
@@ -6,54 +6,67 @@
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+//console.log(colors.push("indigo"))
 
-// a) Your answer:
+// a) Your answer: 
+ //   5
 // b) Verify and explain:
+ //   The answer is 5 because the .push built-in method adds a value to the end of an array and returns the position of that value. 
```

Comment:
Great job for answering 5, this one is tricky!  The position or index of the pushed value would actually be 4, but the length of the array is 5 and is the return of the .push() method.